src/callrpc.cpp

```diff
@@ -189,8 +189,11 @@ bool IsConfirmedBitcoinBlock(const uint256& hash, int nMinConfirmationDepth)
         UniValue result = find_value(reply, "result");
         if (!result.isObject())
             return false;
-        result = find_value(result.get_obj(), "confirmations");
-        return result.isNum() && result.get_int64() >= nMinConfirmationDepth;
+
+        UniValue confirmations = find_value(result.get_obj(), "confirmations");
+		UniValue nTx = find_value(result.get_obj(), "nTx");
```

Comment:
spacing

Reply:
Ah. Didn't configure Vim to expand tabs for C++ I think.